modules/light-clients/07-tendermint/misbehaviour_handle.go

```diff
@@ -68,30 +86,6 @@ func (cs ClientState) CheckForMisbehaviour(ctx sdk.Context, cdc codec.BinaryCode
 // to misbehaviour.Header2
 // Misbehaviour sets frozen height to {0, 1} since it is only used as a boolean value (zero or non-zero).
```

Comment:
Is there anything we should update in the godoc of this function after this change?